src/cpp/src/continuous_batching_pipeline.cpp

```diff
@@ -81,8 +82,12 @@ class ContinuousBatchingPipeline::Impl {
 
         apply_paged_attention_transformations(model, device_config);
 
-        ov::InferRequest infer_request = core.compile_model(model, device_config.get_device(), plugin_config).create_infer_request();
-
+        auto compiled_model = core.compile_model(model, device_config.get_device(), plugin_config);
+        read_properties([compiled_model](const std::string& key) {
+            return compiled_model.get_property(key); },
+            m_model_config_namevalues);
```

Comment:
Can we simplify this? Instead of lambda I think we could have a dedicated read method that takes compiled model as a parameter. I don't think we need this level of generalization at this point.